static/js/stream_list.js

```diff
@@ -529,6 +530,21 @@ export function set_event_handlers() {
             toggle_filter_displayed(e);
         });
 
+    $("#streams_inline_cog")
```

Comment:
We really should to change this selector... I believe it says "cog" because we used to have a cog wheel icon there... we've since changed to a plus icon. I think `streams_inline_icon` would be an acceptable new name in case we decide to change to something else in the future.

Reply:
Thanks, I have updated the PR.